src/scss/_main.scss

```diff
@@ -443,6 +442,14 @@ nav {
   margin: 0 8px 0 8px;
 }
 
+.row .message {
+  font-size: 16px;
```

Comment:
kinda picky, but we're trying to use `rem` rather than `px` for font-size (although some of the older style is still in px and needs to be updated) but good to get in the habit
I think you can satisfy
```font-size: 16px;```
with
```font-size: 1rem;```

Reply:
more info: https://engageinteractive.co.uk/blog/em-vs-rem-vs-px

Reply:
Fixed. Should I make this conversion to the other uses of px?